### DISCUSSION TITLE
Process of applying for Commercial API

### DISCUSSION THREAD

**Discussion by erichlin1** [etsy/open-api#1219] [Q&A] [answered]
Hi all,

I am considering on opening my application that I developed for my shop, for sellers. Ideally, I would like to know if Etsy will approve of this app before starting the development process. From my understanding, it does satisfy all the [criteria](https://www.etsy.com/developers/documentation/launching/get_full_access) set by Etsy, but I have heard stories of folks getting denied regardless. In this case, could I just create my marketing website with content that goes in detail what my app will do, then apply for Commerical API?

Any recommendations or insights would be great, thank you!

**Comment by simonj** [comment, marked as answer]
They said to me they needed a landing page that explain what your application do. 
I also had some GUI that I made a video showing them also

**Reply by erichlin1** [reply]
Great, thanks for that!

**Reply by erichlin1** [reply]
Hey @simonj Did they mention legal document requirements like 'privacy policy' and 'cookies policy' on this landing page?

**Reply by simonj** [reply]
No not at all
